Dockerfile

```diff
@@ -307,6 +309,11 @@ COPY whitelist.json /lib/whitelist.json
 WORKDIR /
 
 
+# Workaround to utilize the global GO_VERSION argument
+# since "COPY --from" doesn't support variables.
+FROM golang:${GO_VERSION}-stretch as golang-ignore
```

Comment:
ok, this is pretty hacky but it seems like the best way without resorting to templating the Dockerfile.